kousa/lib/broth/message/user/get_info.ex

```diff
@@ -43,6 +44,7 @@ defmodule Broth.Message.User.GetInfo do
       field(:numFollowers, :integer)
       field(:online, :boolean)
       field(:lastOnline, :utc_datetime_usec)
+      field(:lastChatMsg, :utc_datetime_usec)
```

Comment:
this shouldn't be stored in the db, look at how we throttle in chat genserver

Reply:
done